Coverage pollutes the test output with these warnings:

CoverageWarning: Module build was previously imported, but not measured (module-not-measured)

The warning is probably related to tox-current-env.  We need try to solve the
warning, but for now we simply disable coverage.

--- build-1.2.2.post1/tox.ini.orig
+++ build-1.2.2.post1/tox.ini
@@ -26,9 +26,7 @@
     PYTHONWARNDEFAULTENCODING = 1
     TEST_STATUS_DIR = {envtmpdir}
 commands =
-    pytest -ra --cov --cov-config pyproject.toml \
-      --cov-report=html:{envdir}/htmlcov --cov-context=test \
-      --cov-report=xml:{toxworkdir}/coverage.{envname}.xml {posargs:-n auto}
+    pytest -ra {posargs:-n auto}
 
 [testenv:fix]
 description = run static analysis and style checks
